.github/ISSUE_TEMPLATE/support-request.yml

```diff
@@ -32,3 +32,7 @@ body:
   attributes:
     label: Additional context
     description: Please attach here any material you may consider useful (pictures, screenshots, logs, etc.).
+- type: textarea
+  attributes:
+    label: How does it affect me?
```

Comment:
I was wondering if ``me`` or ``you`` is the most appropriate. For the person opening the issue, the question "How does it affect me?" might look like "How does it affect icub-tech?"

Reply:
Good point!
Let me amend it!